packages/tokens/src/figma-tokens/_docs.json

```diff
@@ -917,14 +955,8 @@
           "type": "text"
         },
         "value": {
-          "title": {
-            "value": "Initial Value",
-            "type": "text"
-          },
-          "description": {
-            "value": "To set the initial value and checked state, use the value attribute on the containing radio group.",
-            "type": "text"
-          }
+          "title": "[object Object]",
```

Comment:
This seems to be kind of wrong. Object Object is often seen in JavaScript when using something like `{ hello: 'world' }`. Maybe check the syntax in Figma?